src/DotNetBoilerplate.Infrastructure/DAL/Handlers/Books/BrowseBooksByBookStoreIdHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using DotNetBoilerplate.Application.Books.DTO;
+using DotNetBoilerplate.Application.Books.Browse;
+using DotNetBoilerplate.Core.Books;
+using DotNetBoilerplate.Shared.Abstractions.Queries;
+using DotNetBoilerplate.Infrastructure.DAL.Contexts;
+using Microsoft.EntityFrameworkCore;
+
+namespace DotNetBoilerplate.Infrastructure.DAL.Handlers.Books;
+
+internal sealed class BrowseBooksByBookStoreIdHandler(
```

Comment:
Ja bym to po prostu nazwał  BrowseBooks i jako opcjonalny argument dodał bookStoreId. Wtedy mógłby msprawdzić if bookStoreId in not null books=books   .Where(x => x.BookStoreId == query.Id)